core/net/ip64/ip64-ipv4-dhcp.c

```diff
@@ -58,10 +61,10 @@ PROCESS_THREAD(ip64_ipv4_dhcp_process, ev, data)
 
   ip64_dhcpc_init(&ip64_eth_addr, sizeof(ip64_eth_addr));
 
-  printf("Inited\n");
+  PRINTF("Inited\n");
 
   ip64_dhcpc_request();
-  printf("Requested\n");
+  PRINTF("Requested\n");
```

Comment:
Maybe we should change the prints to something more meaningful i.e `IP64: Initited`, etc